### DISCUSSION TITLE
Não é muito bem uma duvida dessa ficha

### DISCUSSION THREAD

**Discussion by Lvkzera** [alyssapiresfernandescefet/openrpg#15] [General]
Eu queria saber se possivelmente você ira fazer uma ficha desse jeito porem pro novo sistema do livro.

**Comment by lucaspiresfernandes** [comment]
Que livro você quer dizer?

**Reply by Lvkzera** [reply]
o livro do rpg de ordem paranormal, o oficial msm, fazer uma ficha de personagem no estilo de calamidade

**Reply by lucaspiresfernandes** [reply]
Não vou fazer. O objetivo do Open RPG é ser uma ferramenta para todos os sistemas, não pretendo criar nada específico para apenas um sistema só.
